### DISCUSSION TITLE
why does i dont get smooth curves

### DISCUSSION THREAD

**Discussion by Selandros** [suchmememanyskill/PrusaSlicer-Ender3-v3-SE-Config#39] [General]
the curves at the and of the print, u have any idé to fix it. do i need to sand it down? 
![image](https://github.com/suchmememanyskill/PrusaSlicer-Ender3-v3-SE-Config/assets/40951984/e18c473b-e5f8-498a-86ad-42cc2b92227c)

**Comment by suchmememanyskill** [comment]
Might be worth using a different top layer infill pattern. Might also be an issue with the filament or printer. Or a hidden issue with my slicer profile(? I get this effect also but way less bad)

**Reply by Selandros** [reply]
it be less visable with higher slice resolution, but then the object is not round anymore, so good topp or round object.. ![image](https://github.com/suchmememanyskill/PrusaSlicer-Ender3-v3-SE-Config/assets/40951984/72a87ee7-3f6a-4013-9d99-ebddb807369f)

**Reply by suchmememanyskill** [reply]
Could you try changing this value to 15%? In the profile it's currently 25%

![image.png](https://github.com/suchmememanyskill/PrusaSlicer-Ender3-v3-SE-Config/assets/38142618/8b339794-4aca-46b4-ab3e-ff5d86880903)

**Reply by Selandros** [reply]
> Could you try changing this value to 15%? In the profile it's currently 25%
> 
>

much better :) 
0 slice
15% 👍

![image](https://github.com/suchmememanyskill/PrusaSlicer-Ender3-v3-SE-Config/assets/40951984/fb222e74-b02d-4dcf-9018-fab457f34095)

**Reply by suchmememanyskill** [reply]
Might change that in the profile itself then; was testing with it today and noticed 15% was optimal
